src/core/WebAuthnP256.ts

```diff
@@ -397,8 +398,9 @@ export function getCredentialRequestOptions(
       challenge: Bytes.fromHex(challenge),
       rpId,
       userVerification,
+      extensions,
```

Comment:
Let's move this up under `challenge` too so alphabetized.

```suggestion
      ...(extensions && { extensions }),
```